npm/lib/node-install.ts

```diff
@@ -9,23 +9,23 @@
 import {
   downloadedBinPath,
   pkgAndSubpathForCurrentPlatform,
-} from "./node-platform";
+} from './node-platform';
```

Comment:
These files have already been formatted with prettier, mostly—it seems like this change is just to change the `'` to `"`?

Reply:
In this discussion, several colleagues said they preferred single quotes, so I decided to apply it throughout the project.

https://github.com/cloudflare/workerd/pull/2543#discussion_r1720089448

Reply:
Fair enough—as long as we're consistent. Worth noting that we use `"` in [workers-sdk](https://github.com/cloudflare/workers-sdk/blob/main/.prettierrc#L3)

Reply:
@jasnell, @anonrig

Reply:
Obvs not a big deal but I prefer double quotes since JSON requires them, and being consistent with workers-sdk would be a nice benefit

Reply:
Quote fight!

Reply:
I'm on team single quotes, but it's not a big deal for me either.

Reply:
We can use different configuration on different folders. We just need to add `.prettierrc` to the folder where we want to have double quotes.

Reply:
I would prefer to have a unified style in all of workerd, but I'm not opposed to per-folder if there is a strong reason